pydantic/validators.py

```diff
@@ -200,6 +201,22 @@ def ordered_dict_validator(v: Any) -> 'AnyOrderedDict':
         raise errors.DictError()
 
 
+def defaultdict_validator(v: Any, field: 'ModelField') -> DefaultDict[Any, Any]:
+    # no test coverage for almost all of this function: ModelField._validate_mapping() always returns dict type.
```

Comment:
pydantic has 100%. That shouldn't change.

If there's code that can't be run, please remove it and leave a comment.